website/source/intro/getting-started/running.html.md

```diff
@@ -134,9 +134,13 @@ replication continues to be attempted until the node recovers. Nomad will
 automatically try to reconnect to _failed_ nodes, allowing it to recover from
 certain network conditions, while _left_ nodes are no longer contacted.
 
-If an agent is operating as a server, a graceful leave is important to avoid
-causing a potential availability outage affecting the
-[consensus protocol](/docs/internals/consensus.html). If a server does
+If an agent is operating as a server, [leave_on_terminate](/docs/agent/configuration/index.html#leave_on_terminate) should only
+be set if the server will never rejoin the cluster again. The default value of `false` for 'leave_on_terminate` and `leave_on_interrupt`
+work well for most scenarios. If Nomad servers are part of an auto scaling group where new servers are brought up to replace
+failed servers, using graceful leave avoids causing a potential availability outage affecting the [consensus protocol](/docs/internals/consensus.html).
+With newer features in [autopilot](/guides/cluster/autopilot.html) in Nomad 0.8, failed or dead servers are removed if they have been replaced.
```

Comment:
As of of Nomad 0.8, Nomad includes Autopilot which automatically removes failed or dead servers. This allows the operator to skip setting `leave_on_terminate`